app/src/models/InventoryValue.ts

```diff
@@ -28,7 +28,8 @@ export interface InventoryValueAttributes {
   subSectorId?: string;
   subCategoryId?: string;
   inventoryId?: string;
-  datasourceId?: string | null;
+  /** @deprecated moved to ActivityValue */
+  datasourceId?: string | null; // TODO remove
```

Comment:
This change can be reverted as the important change was already merged and it is no longer deprecated.

Reply:
Are we blocking on this or can we merge?